hasura/graphql-engine
# ra-data-hasura: add filters to GET_* types

### Description
Set filters for some type. The filters were not being set, therefore they any filters added to list type components were being ignored. This was done for GET_LIST, GET_MANY and GET_MANY_REFERENCE types.

### Affected components 
- ra-data-hasura

### Solution and Design
Created `addFilter` function to add filters to `args.where`. 

### Steps to test and verify
Add a `filter` prop to a`ReferenceManyField` component. You will notice the filter has no effect. This change will apply the filter as expected.

```js
<ReferenceManyField
  filter={{ type: 'geofence' }}
  reference="event"
  target="account_id"
>
```
